main.go

```diff
@@ -679,7 +680,16 @@ func main() {
 		log.Printf("[INFO]  : Debug mode : %v", config.Debug)
 	}
 
-	if err := http.ListenAndServe(fmt.Sprintf("%s:%d", config.ListenAddress, config.ListenPort), nil); err != nil {
+	server := &http.Server{
+		Addr: fmt.Sprintf("%s:%d", config.ListenAddress, config.ListenPort),
+		// Timeouts
+		ReadTimeout:       60 * time.Second,
```

Comment:
we might expose those configs to be configurable, i can do in a follow up

Reply:
I agree